.github/workflows/ci.yml

```diff
@@ -21,18 +21,15 @@ jobs:
           - Windows
           - macOS
         python-version:
-          - 3.8
+          - 3.9
           - 2.7
           - 3.5
           - 3.6
           - 3.7
+          - 3.8
         pip-version:
           - "latest"
           - "20.1"
-        include:
-          - os: Ubuntu
-            python-version: 3.9-dev
```

Comment:
Maybe test `3.10-dev` now? Deadsnakes supports it.

Reply:
Done!